README.md

```diff
@@ -60,10 +63,53 @@ The bundle will deploy:
 - 1 `coordinator` unit (`tempo-coordinator-k8s` charm)
 - 1 `s3-integrator` unit (`s3-integrator` charm)
 
-[note]
+[!NOTE]  
 The default number of worker units is an odd number to prevent split-brain situations, and greater than 1 for the deployment to be HA.
-[/note]
 
+### Minimal microservices deployment
+The bundle is able to deploy Tempo in its [microservices mode](https://grafana.com/docs/tempo/latest/setup/deployment/#microservices-mode), which runs each one of the required roles in distinct processes. Each required role is assigned to one worker unit to achieve the minimal deployment for the cluster to be considered consistent. Minimal microservices mode is the way to deploy Grafana Tempo in a distributed architecture without high availability.
+
+Add `--mode` to deploy `tempo-bundle` in `minimal-microservices` mode.
+
+```shell
+# render bundle with "edge" charms
+tox -e render-edge -- --mode=minimal-microservices
+juju deploy ./bundle.yaml --trust
+```
+
+The bundle will deploy:
```

Comment:
please rephrase this listing following the same format as suggested for the previous one (include app names). I'd call the worker nodes `tempo-<role>`, omitting the 'worker'